tests/elements-regression/tests/elements-regression.test.ts

```diff
@@ -36,7 +36,7 @@ test.describe( 'Elementor regression tests with templates for CORE', () => {
 		'tabs',
 		'video',
 		'spacer',
-		'text_path',
+		// 'text_path',
```

Comment:
@davseve Do you have a task to unskip this?

Reply:
Yes, of course https://elementor.atlassian.net/browse/ED-17796